src/i18n/locales/en.json

```diff
@@ -806,6 +806,15 @@
     "index_general_description": "Dependency Track uses Apache Lucene to enable full-text search on various entities like projects or vulnerabilities.",
     "index_use_cases": "The full-text search feature is principally used for the search API (i.e. all the indexes) and internal analyzer fuzzy matching on CPE (i.e. the vulnerable software index).",
     "index_issues_description": "The lucene indexes can degrade or drift from Dependency Track database over time. Even though DT does its best to minimize the drift, the administrative features below are provided to check or restore the indexes if need be. It must be used with caution.",
+    "index_risk_score_description": "Dependency Track calculates the risk score of each project as a weighted severity score using the following calculation as default: ",
+    "index_risk_score_calc": "((critical * 10) + (high * 5) + (medium * 3) + (low * 1) + (unassigned * 5))",
+    "index_risk_score_weighting_description": "You can customize the weighting of each severity to customize the risk score.",
```

Comment:
Any particular meaning behind the `index_` prefix?

Reply:
i think i copied it from a similar line unnecessarily, removed the "index_" from these